constitution.md

```diff
@@ -0,0 +1,232 @@
+# Chisel Working Group (CWG) Provisional Constitution
+
+### Table of Contents
+* [Relationship to CHIPS Alliance](#relationship-to-chips-alliance)
+* [Mission Statement](#mission-statement)
+* [Members](#members)
+  * [Technical Advisory Committee](#technical-advisory-committee)
+  * [Administrator](#administrator)
+  * [Reviewers](#reviewers)
+  * [Developers](#developers)
+  * [Volunteers](#volunteers)
+* [Collaboration Policy](#collaboration-policy)
+  * [Communication Channels](#communication-channels)
+  * [Development Policy](#development-policy)
+* [Amending the Constitution](#amending-the-constitution)
+* [TAC Voting](#tac-voting)
+* [Code of Conduct (Contributors)](#code-of-conduct-contributors)
+  * [Our Pledge](#our-pledge)
+  * [Our Standards](#our-standards)
+  * [Our Responsibilities](#our-responsibilities)
+  * [Scope](#scope)
+  * [Enforcement](#enforcement)
+  
+## Relationship to CHIPS Alliance
+
+The Chisel Working Group is a member of CHIPS Alliance and subject to the bylaws established by
+the CHIPS Alliance Board.
+
+## Mission Statement
+The Chisel ecosystem is a stable, user-friendly platform for production-quality RTL generator development, 
```

Comment:
```suggestion
The Chisel ecosystem is intended to provide a stable, user-friendly platform for production-quality RTL generator development, 
```

Reply:
Is this satisfactory to leave, @edwardcwang ?

Reply:
My personal take is that this comes across a little bit as self-congratulatory - e.g. "mission accomplished, we are the best". Though it's also fine with me to leave it as-is.